pkgs/by-name/ko/koboldcpp/package.nix

```diff
@@ -43,23 +47,25 @@ let
 in
 effectiveStdenv.mkDerivation (finalAttrs: {
   pname = "koboldcpp";
-  version = "1.68";
+  version = "1.69.1";
 
   src = fetchFromGitHub {
     owner = "LostRuins";
     repo = "koboldcpp";
     rev = "refs/tags/v${finalAttrs.version}";
-    sha256 = "sha256-zqRlQ8HgT4fmGHD6uxxa2duZrx9Vhxd+gm1XQ7w9ay0=";
+    sha256 = "sha256-808HyqrB9/cFJU7nq7Cm5Fm4OFPML4oQtnv0hystwAg=";
   };
 
   enableParallelBuilding = true;
 
+  LD_LIBRARY_PATH = [ addDriverRunpath.driverLink ];
```

Comment:
This is only set at build time, when `${driverLink}/lib` isn't available because of the sandbox.

Reply:
I think I'll need a bit of a helping hand on this here, what exactly do I need to do here?

I prefixed the `LD_LIBRARY_PATH` in `postFixup`, is that enough?

```nix
${lib.optionalString config.cudaSupport ''
  \
    --prefix LD_LIBRARY_PATH: "${lib.makeLibraryPath finalAttrs.LD_LIBRARY_PATH}"
''}
```

Reply:
`makeLibraryPath` is enough yes, it does add the `/lib` suffix and also chooses the correct output for derivations